tests/Dfe.PlanTech.Application.UnitTests/Submissions/Queries/GetAllAnswersForLatestSubmissionQueryTests.cs

```diff
@@ -0,0 +1,127 @@
+    {
+        const string section = "broadband-connection";
+        const int establishmentId = 123;
+        
+        var responses = new List<Response>
+        {
+            new Response()
+            {
+                AnswerId = 1,
+                QuestionId = 1,
+                UserId = 1,
+                SubmissionId = 1,
+                Maturity = "low",
+                DateCreated = DateTime.UtcNow.AddHours(-1),
+                DateLastUpdated = null
+            },
+            new Response()
+            {
+                AnswerId = 2,
+                QuestionId = 2,
+                UserId = 2,
+                SubmissionId = 1,
+                Maturity = "medium",
+                DateCreated = DateTime.UtcNow,
+                DateLastUpdated = null
+            },
+            new Response()
+            {
+                AnswerId = 3,
+                QuestionId = 1,
+                UserId = 1,
+                SubmissionId = 2,
+                Maturity = "low",
+                DateCreated = DateTime.UtcNow,
+                DateLastUpdated = null
+            },
+            new Response()
+            {
+                AnswerId = 4,
+                QuestionId = 2,
+                UserId = 2,
+                SubmissionId = 2,
+                Maturity = "medium",
+                DateCreated = DateTime.UtcNow,
+                DateLastUpdated = null
+            }
+        };
+
+        var submissions = new List<Submission>
+        {
+            new Submission()
+            {
+                SectionId = section,
+                SectionName = "Bob",
+                EstablishmentId = establishmentId,
+                Completed = true,
+                DateCreated = DateTime.UtcNow,
+            },
+            new Submission()
+            {
+                SectionId = section,
+                SectionName = "Bob",
+                EstablishmentId = establishmentId,
+                Completed = true,
+                DateCreated = DateTime.UtcNow,
+            }
+        };
+
+       
+
+        var answers = new List<Answer>
+        {
+            new Answer()
+            {
+                Id = 1,
+                AnswerText = "Answer 1",
+                ContentfulRef = "Ref 1"
+            },
+            new Answer()
+            {
+                Id = 2,
+                AnswerText = "Answer 2",
+                ContentfulRef = "Ref 1"
+            },
+            new Answer()
+            {
+                Id = 3,
+                AnswerText = "Answer 3",
+                ContentfulRef = "Ref 1"
+            }
+            ,    new Answer()
+            {
+                Id = 4,
+                AnswerText = "Answer 4",
+                ContentfulRef = "Ref 1"
+            }
+        };
+        
+        var options = new DbContextOptionsBuilder<PlanTechDbContext>().UseInMemoryDatabase(databaseName: Guid.NewGuid().ToString()).Options;
```

Comment:
This is fine here, but be careful of this in future. The `InMemoryDatabase` won't match functionality of the SQL Server one, so depending on what you execute/do it might return unexpected results.